src/Components/Atoms/AuthButton/AuthButtonPropsType.ts

```diff
@@ -0,0 +1,3 @@
+export interface AuthButtonPropsType{
+	text:string;
```

Comment:
Keep it just Button not AuthButton as we will not create multiple components on the basis of various buttons.